.github/dependabot.yml

```diff
@@ -1,148 +1,129 @@
 version: 2
```

Comment:
I discovered that Dependabot will actually handle all the packages in the monorepo so we don't need to specify them individually. See https://github.com/ilios/frontend/blob/master/.github/dependabot.yml

Reply:
consolidated listed dependencies and removed package-specific configuration. 

:white_check_mark: